index.html

```diff
@@ -0,0 +1,211 @@
+          <a class="navbar-brand logo d-none d-md-block" href="index.html">
+            <img src="./assets/img/full-logo.svg" alt="orange-react-dev-conference">
+          </a>
+          <button class="navbar-toggler border-0 px-0" type="button" data-bs-toggle="collapse"
+            data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
+            aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse w-100 py-5 py-md-0" id="navbarSupportedContent">
+            <ul class="navbar-nav ms-auto mb-2 mb-lg-0 list-unstyled ms-0 ps-0 d-flex align-items-center">
+              <li class="nav-item ms-md-3 py-2 py-md-0">
+                <a class="nav-link active" aria-current="page" href="index.html">Home</a>
+              </li>
+              <li class="nav-item ms-md-3 py-2 py-md-0">
+                <a class="nav-link" aria-current="page" href="about.html">About</a>
+              </li>
+              <li class="nav-item ms-md-3 py-2 py-md-0">
+                <a class="nav-link" aria-current="page" href="#">Program</a>
+              </li>
+              <li class="nav-item ms-md-5 mt-3 mt-md-0">
+                <a class="nav-link emphasized-link px-3" aria-current="page" href="#">CC Campaign</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </nav>
+  </header>
+  <main>
+    <section class="hero pt-5">
+      <div class="container py-md-5 mt-md-5">
+        <div class="row py-5">
+          <div class="col">
+            <span class="h1">Welcome Developers To</span>
+            <div class="emphasis-header d-block my-3">
+              <h1 class="h1">Orange React <br />Developer Conference <br />2022</h1>
+            </div>
+            <p class="description d-block p-3">
+              The only constant in life is change.” Our engineering friend Heraclitus once said - and that's exactly
+              why:
+              Security and the legal framework have first priority!
+            </p>
+            <div class="emphasis-subtitle py-3">
+              <h2 class="h2 dark-text">2022.03.07(Mon) - 9(Wed)</h2>
+              <p class="h5 dark-text">@Landmark Events Center, Lagos</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="program dark-bg-2 text-white py-5">
+      <div class="container py-md-5 py-3">
+        <div class="row">
+          <h3 class="section-title light">Main Program</h3>
+        </div>
+        <div class="row py-md-3">
+          <div class="program-items">
+            <div class="program-item">
+              <img class=" mx-md-auto" src="./assets/img/icons/talk.png" alt="speaker-icon">
+              <h4 class="title">Lectures</h4>
+              <p>Listen to the speakers from various countries about the messages of sharing and opening.</p>
+            </div>

```

Comment:
- [ ] Please let's make use of the `<article></article>` element here instead of the generic `div` for semantic reasons since we are actually adding articles here. That goes the same for all articles on this page.